Automattic/vip-go-mu-plugins
# FORNO-1668: Add Prometheus user stats collector

## Description

Adds a Prometheus collector to count users by role and 2FA status. Initially limited to `develop` and `staging` environments.

## Changelog Description

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

1. Check out PR
1. Enable the `prom-user-collection` feature for your test environment (e.g. by adding `local` [here](https://github.com/Automattic/vip-go-mu-plugins/blob/e324084b849ceae193f45fe6a1e383ad21c3f3b4/lib/feature/class-feature.php#L60-L63))
1. Run the cron to ensure metrics have been processed `wp cron event run vip_aggregated_cron_hourly`
1. Visit the site to trigger collection
2. Vist `<siteurl>/.vip-prom-metrics` to ensure the metrics are present
